acme/acme/client.py

```diff
@@ -624,7 +624,32 @@ def _send_request(self, method, url, *args, **kwargs):
         kwargs.setdefault('headers', {})
         kwargs['headers'].setdefault('User-Agent', self.user_agent)
         kwargs.setdefault('timeout', self._default_timeout)
-        response = self.session.request(method, url, *args, **kwargs)
+        try:
+            response = self.session.request(method, url, *args, **kwargs)
+        except requests.exceptions.RequestException as e:
+            """ Regex for parsing error messages"""
```

Comment:
Instead of terminating each line, please use the ability of Python's triple-quote """ to span multiple lines, and remove the initial space on the first line:

```
"""First line
   Second line
"""
```

Also, good comments should explain "why" rather than "what" or "how." For instance, here we would say something like "The requests library emits exceptions with a lot of extra text. We parse them with a regexp to raise a more readable exceptions."